release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -163,6 +163,13 @@ As a cluster administrator, you can configure `internalTrafficPolicy=Local` on a
 
 For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].
 
+[id="ocp-4-11-alb-operator-support"]
+==== Support for AWS Load Balancer Operator (TP)
+
+As a cluster administrator, you can install AWS Load Balancer Operator from the OperatorHub by using {product-title} web console or CLI. The AWS Load Balancer Operator is in technology preview.
```

Comment:
```suggestion
As a cluster administrator, you can install the AWS Load Balancer Operator from OperatorHub using the {product-title} web console or CLI. The AWS Load Balancer Operator is available as a Technology Preview.
```

Reply:
I think you also need to add the AWS Load Balancer Operator in Technology Preview tracker tabel.
 https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

Reply:
@darshan-nagaraj IMO, The Operator is not a technology preview, but it has reached TP status. Similarly, it is not a Technology Preview, but it is in TP _status_.

Reply:
Addressed other suggestions, thanks for the review

Reply:
> I think you also need to add the AWS Load Balancer Operator in Technology Preview tracker tabel. https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

Addressed